### PR TITLE
updating cards on build a backend

### PR DESCRIPTION
**Pull request by offlineprogrammer**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/[platform]/build-a-backend/troubleshooting/index.mdx [hunk 1]
```diff
@@ -3,7 +3,8 @@ import { getChildPageNodes } from '@/utils/getChildPageNodes';
 
 export const meta = {
   title: 'Troubleshooting',
-  description: 'Troubleshooting',
+  description:
+    'Recommendations on how to resolve some of the common issues & problems that you might face when using Amplify',
```
Comment (heatheramz) [inline src/pages/[platform]/build-a-backend/troubleshooting/index.mdx:7]:
```suggestion
    'Recommendations on how to resolve some of the common issues and problems that you might face when using Amplify',
```

**Review by heatheramz — COMMENTED** [review]
One minor change as we are trying to not use ampersands when possilbe.

File: src/pages/[platform]/prev/build-a-backend/sever-side-rendering/index.mdx [hunk 2]
```diff
@@ -2,9 +2,19 @@ import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
 
 export const meta = {
   title: 'Server-Side Rendering',
-  description: 'Get started with Server-Side Rendering (SSR)',
-  platforms: ['javascript', 'angular', 'nextjs', 'react', 'vue'],
-    route: '/[platform]/build-a-backend/server-side-rendering'
+  description: 'Use Amplify categories APIs from Next.js server-side runtimes.',
+  platforms: [
+    'android',
```
Comment (heatheramz) [inline src/pages/[platform]/prev/build-a-backend/sever-side-rendering/index.mdx:7]: @offlineprogrammer we'll need to remove the platforms that don't use SSR here

**Review by heatheramz — APPROVED** [review]
LGTM

**Review by renebrandel — APPROVED** [review]

**Pull request merged**